cscs-checks/tools/jupyter/jupyter_tests.py

```diff
@@ -8,15 +8,35 @@
 
 
 @rfm.simple_test
-class JupyterHubSubmitTest(rfm.RunOnlyRegressionTest):
-    def __init__(self):
-        self.valid_systems = ['daint:jupyter_gpu', 'daint:jupyter_mc',
-                              'dom:jupyter_gpu', 'dom:jupyter_mc']
-        self.valid_prog_environs = ['*']
-        self.sourcesdir = None
-        self.executable = 'hostname'
-        self.time_limit = '1m'
-        self.max_pending_time = '7m'
+class jupyterhub_submit_test(rfm.RunOnlyRegressionTest):
+    valid_systems = ['daint:jupyter_gpu', 'daint:jupyter_mc',
+                     'dom:jupyter_gpu', 'dom:jupyter_mc']
+    valid_prog_environs = ['*']
+    sourcesdir = None
+    executable = 'hostname'
+    time_limit = '1m'
+    max_pending_time = '7m'
+    tags = {'production', 'post-maintenance', 'health'}
+    maintainers = ['RS', 'TR']
+
+    @rfm.run_before('sanity')
+    def set_sanity_check(self):
         self.sanity_patterns = sn.assert_found(r'nid\d+', self.stdout)
-        self.tags = {'production', 'maintenance'}
-        self.maintainers = ['RS', 'TR']
+
+
+@rfm.simple_test
+class jupyterhub_api_test(rfm.RunOnlyRegressionTest):
+    descr = 'Check JupyterHub server status and version'
+    valid_systems = ['daint:jupyter_gpu', 'daint:jupyter_mc',
+                     'dom:jupyter_gpu', 'dom:jupyter_mc']
+    valid_prog_environs = ['*']
```

Comment:
Is there any reason why we have `*` instead of `builtin`?

Reply:
@ChristopherBignamini Can you reply to this?

Reply:
My apologies. No particular reason for having *, let's change it to builtin.